README.md

```diff
@@ -79,8 +79,8 @@ Using [Zinit](https://github.com/zdharma-continuum/zinit):
 
 2. Load the plugin in `~/.zshrc`:
 
-        zinit load 'zsh-users/zsh-history-substring-search
-        zinit ice wait atload'_history_substring_search_config'
+        zinit load 'zsh-users/zsh-history-substring-searchf
+        zinit ice wait atload '_history_substring_search_config'
```

Comment:
not a typo, I believe - `zinit` requires there to be no space (they're supposed to be named-arguments, though as an argument parsing scheme it seems kinda risky :)